mmaction/datasets/transforms/loading.py

```diff
@@ -265,6 +265,182 @@ def __repr__(self):
         return repr_str
 
 
+@TRANSFORMS.register_module()
+class UniformSampleFrames(BaseTransform):
+    """Uniformly sample frames from the video.
+
+    To sample an n-frame clip from the video. UniformSampleFrames basically
+    divide the video into n segments of equal length and randomly sample one
+    frame from each segment. To make the testing results reproducible, a
+    random seed is set during testing, to make the sampling results
+    deterministic.
+
+    Required keys are "total_frames", "start_index" , added or modified keys
```

Comment:
It will be more clear to list required/added keys in separate lines like

Required keys:

    - KeyA
    - KeyB
    - ...